### ISSUE TITLE
F6 key needed to cycle through different panes

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6882]
`F6` and `Shift+F6` have been widely used in many applications to cycle through diferent panes.

VSCode and Slack, for example, have utilized these keys to provide keyboard users to quickly jump over to next and previous panes.

Slack has added `CTTRL` modifier key to avoid the conflict against go-to-addessbar in their web interface like `CTRL+F6`.

We may want to have this kind of key for RStudio Desktop and Server as well.
